app/assets/stylesheets/groups.scss

```
@@ -7,6 +7,12 @@
   font-weight: normal;
 }
 
+.null-text {
+  color: $primary-green;
```

```suggestion
  color: $iframe-container-grey;
```
Try this color not primary for the message!